code/datums/keybinding/xeno.dm

```diff
@@ -694,6 +694,24 @@
 	keybind_signal = COMSIG_XENOABILITY_SCREECH
 	hotkey_keys = list("E")
 
+/datum/keybinding/xeno/heal_screech
```

Comment:
perhaps look at setting it like defiler's neurogas selection, where you have a second button to select screech type https://github.com/tgstation/TerraGov-Marine-Corps/blob/731b6f7b952bc3c565b058629a613e694cd6ad1b/code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm#L357 Or like plant weeds, where it is hidden inside the button's selection https://github.com/tgstation/TerraGov-Marine-Corps/blob/731b6f7b952bc3c565b058629a613e694cd6ad1b/code/modules/mob/living/carbon/xenomorph/abilities.dm#L89